main.py

```diff
@@ -109,8 +114,13 @@ def _request_flow_list(self, switch):
                                                              switch)
             xid_ports = of_core_v0x04_utils.request_port_stats(self.controller,
                                                                switch)
-            self._multipart_replies_xids[switch.id] = {'flows': xid_flows,
-                                                       'ports': xid_ports}
+            xid_tables = of_core_v0x04_utils.update_table_list(self.controller,
```

Comment:
Can you rename `_request_flow_list` as `_request_stats`? and `request_flow_list` as `request_stats`? These have evolved and they're requesting multiple stats instead of just flow. It's fine to keep in the same func as they're all closely related but let's keep the name coherent too for readability.

Reply:
You are right!

Reply:
@gretelliz can you double check this refactoring? `request_list` is meant to be `request_stats` and `_request_list` is meant to be `_request_stats`, thanks. As it is, `request_list` is confusing and too vague.